sales.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title> Salmon Cookies! </title>
```

Comment:
please also check the other most recent submit, it has changes in index.html as well